docs/guide/querying.rst

```diff
@@ -252,6 +252,24 @@ and provide the pipeline as a list
 
 .. versionadded:: 0.23.2
 
+Update with Array Operator
+--------------------------------
+It is possible to update specific value in array by use array_filters (arrayFilters) operator.
+This is done by using ``__raw__`` keyword argument to the update method and provide the arrayFilters as a list.
+
+`Update with Array Operator <https://www.mongodb.com/docs/manual/reference/operator/update/positional-filtered->`_
+::
+
+    # assuming an initial 'tags' field == ['test1', 'test2', 'test3']
+    Page.objects().update(__raw__=
```

Comment:
(previous review was wrong here due to misunderstanding on my end) - We don't want "array_filters" in the `__raw__` dict, we want it provided as plain function argument

```
# assuming an initial 'tags' field == ['test1', 'test2', 'test3']
Page.objects().update(
    __raw__={"$set": {"tags.$[element]": "test11111"}},
    array_filters=[{"element": {"$eq": "test2"}}],
)

# updated 'tags' field == ['test1', 'test11111', 'test3']
```

Reply:
Fixed